tldw_Server_API/app/core/AuthNZ/pg_migrations_extra.py

```
@@ -2883,6 +2902,33 @@ async def ensure_org_provider_secrets_pg(pool: DatabasePool | None = None) -> bo
         return False
 
 
+async def ensure_org_stt_settings_pg(pool: Any | None = None) -> bool:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `pool`

(ANN401)